Microsoft.Dynamics365.UIAutomation.Api.UCI/WebClient.cs

```diff
@@ -1903,6 +1906,19 @@ internal BrowserCommandResult<bool> SetValue(string field, string value)
             });
         }
 
+        private void SetInputValue(IWebDriver driver, IWebElement input, string value, TimeSpan? thinktime = null)
+        {
+            input.SendKeys(Keys.Control + "a");
+            input.SendKeys(Keys.Backspace);
+            driver.WaitForTransaction();
+
+            if (string.IsNullOrWhiteSpace(value))
+                return;
+
+            input.SendKeys(value, true);
+            driver.WaitForTransaction();
+            ThinkTime(thinktime ?? 3.Seconds());
+        }

```

Comment:
this method is used to enter the One Time Code, but will be also used in future implementations.